tools/run_tests/performance/scenario_config.py

```diff
@@ -236,6 +252,19 @@ def scenarios(self):
           server_core_limit=1, async_server_threads=1,
           secure=secure)
 
+      yield _ping_pong_scenario(
+          'cpp_generic_async_streaming_single_channel_throughput_%s' % secstr, rpc_type='STREAMING',
```

Comment:
I think the "throughput"  will get confused with  QPS throughput, but I don't really have a better name right now.